lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
+Helper functions to get the elasticsearch specific configurations
+"""
+
+from configparser import NoSectionError, NoOptionError
+
+
+def get_es_host(config):
+    try:
+        return config.get("elasticsearch", "host")
+    except (NoOptionError, NoSectionError):
+        return ""
+
+
+def get_es_port(config):
+    try:
+        return config.get("elasticsearch", "port")
+    except (NoOptionError, NoSectionError):
+        return ""
```

Comment:
Are these actually used anywhere other than `get_es_url`?? It doesn't seem to me that they ought to be; in which case when don't we just elevate the URL to an `@property` on the config object in the first place and skip all this?

Reply:
These are elasticsearch properties and are very specific to the Elasticsearch modules we have and thus I removed it from `PbenchServerConfig` `@properties` and added it here as a common helper function that can be used by number of future query modules. `es_host` and `es_port` are not used independently anywhere else (at least yet) but having a separate function helps to know whether the port information is missing or the host information from the config file.